src/azure-cli/azure/cli/command_modules/resource/tests/latest/external_modules.bicep

```diff
@@ -0,0 +1,6 @@
+module helloWorld 'br/public:samples/hello-world:1.0.1' = {
+  name: 'helloWorld'
+  params: {
+    name: 'Bicep developers'
+  }
+}
```

Comment:
Could you move the file which the file type not defined in [package_data](https://github.com/Azure/azure-cli/blob/dev/scripts/ci/build.sh#L142) to `tests/latest/data` folder?

Reply:
Moved.

Reply:
We now enforce test data to be under `/data`, so that it can be included when building the test package (https://github.com/Azure/azure-cli/pull/22148).